libsplinter/Cargo.toml

```diff
@@ -90,6 +90,7 @@ experimental = [
     "biome-credentials",
     "biome-key-management",
     "biome-notifications",
+    "biome-refresh-tokens",
```

Comment:
Should this be separated into a separate commit, since this feature guard appears to have been used before this commit?

Reply:
Moved commit earlier in history, so this is introduced earlier